packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -687,11 +697,13 @@ Starting with Mermaid version 9.4, you can use an alternate renderer named elk.
 The _elk_ renderer is an experimenal feature.
 You can change the renderer to elk by adding this directive:
 
-```
+```elk
```

Comment:
This is not elk language.

Reply:
I know, just rendered more nicely. Hack removed.